device/dhcp/servers/v4/dhcpv4serverLeases.yaml

```diff
@@ -0,0 +1,76 @@
+components:
+  schemas:
+    DhcpServerV4.Leases:
+      description: >-
+        Configuration for DHCPv4 Leases.
+      type: object
+      x-status:
+        status: under_review
+      properties:
+        lease_time: 
+          description: >-
+            The Life Time length in seconds that is assigned to a lease if the requesting DHCP client does not specify a specific expiration time.
+          type: integer
+          format: uint32
+          minimum: 300
+          maximum: 30000000
+          default: 86400
+          x-field-uid: 1
+        start_pool_address:
+          description: >-
+            The IP address of the first lease pool.
+          type: string
+          format: ipv4
+          default: 100.1.1.1
+          x-field-uid: 2
+        prefix:
+          description: >-
+            The IPv4 network prefix length to be applied to the address. 
+          type: integer
+          format: uint32
+          default: 24
+          maximum: 32
+          x-field-uid: 3
+        pool_size:
+          description: >-
+            The total number of addresses in the pool.
+          type: integer
+          format: uint32
+          default: 1
+          minimum: 1
+          x-field-uid: 4
+        step:
+          description: >-
+            Increments the network address prefixes within a pool range  where multiple
+            addresses are present. The value is incremented according to the Prefix
+            Length and Step.
+          type: integer
+          format: uint32
+          default: 1
+          minimum: 1
+          x-field-uid: 5
+        router_address:
+          description: >-
+            The Router address advertised by the DHCPv4 server in Offer and Ack messages.
+          type: string
+          format: ipv4
+          x-field-uid: 6
+        primary_dns_server:
+          description: >-
+             The primary DNS server address that is offered to DHCP clients that request this information through a TLV option.
+          type: string
+          format: ipv4
```

Comment:
Any field without a default either make it optional with required or required itself. Else language default will come to the impl.